### ISSUE TITLE
Popovers/tooltips in dynamic UI

### ISSUE THREAD

**Issue by gadenbuie** [rstudio/bslib#708]
Hi @gadenbuie and @cpsievert, 

I mainly track activity of this repo to see any future features in advance and noticed the work on the popovers/tooltips, it is worth noting that when using the bootstrap tooltips they can be a bit erratic when they are displayed via a renderUI that updates regularly, 

```r
library(shiny)
# devtools::install_github("rstudio/bslib@popovers")
library(bslib)

ui <- page_sidebar(
  title = "My dashboard",
  sidebar = "Sidebar",
  "Main content area",
  shiny::uiOutput("tooltip")
)

server <- function(input, output) {

  timer <- shiny::reactiveTimer(2000)

  output$tooltip <- shiny::renderUI({
      timer()
      tags$div(
        style = "
        width: fit-content; 
        height: 30px; 
        border: 1px solid black; 
        padding: 5px;",
        bslib::tooltip(
          span("Card title ", 
          bsicons::bs_icon("question-circle-fill")),
          "Additional info",
          placement = "auto",
          options = list(
            trigger = "hover"
          ),
          id = "my-tooltip"
        )
      )
  }) 
}

shiny::shinyApp(ui, server)
```

When hovering on top of it during a renderUI refresh the tooltip will jump around, maybe there is a nice way to destroy tooltips contained inside uiOutputs just before they refresh? 

Best, 
Michael

_Originally posted by @michaelhogersnplm in https://github.com/rstudio/bslib/pull/702#issuecomment-1649915329_

**Comment by gadenbuie** [comment]
Thanks for the report @michaelhogersnplm and for testing out the new features. I moved your comment into a new issue so we can track work on in it separately from the popovers PR.

**Comment by cpsievert** [comment]
Thanks for the heads-up @michaelhogersnplm. Hopefully we'll improve this situation, but in the meantime, it looks like you can improve the behavior somewhat by specifying `options = list(animation = FALSE)`

**Comment by gadenbuie** [comment]
@cpsievert just to make sure you see it, it turns out we can fix this in #702 by disposing the tooltip/popover when they're removed from the DOM. https://github.com/rstudio/bslib/pull/702#discussion_r1273709728
